deployment/crawler-legacy-deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    io.kompose.service: crawler-legacy
+  name: crawler-legacy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: crawler-legacy
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      annotations:
+        kompose.cmd: kompose convert -f ../docker-compose.yml.dev
+        kompose.version: 1.22.0 (955b78124)
+      creationTimestamp: null
+      labels:
+        io.kompose.service: crawler-legacy
+    spec:
+      containers:
+        - args:
+            - monocle
+            - --elastic-conn
+            - elastic:9200
+            - crawler
+            - --config
+            - /etc/monocle/config.yaml
+          env:
+            - name: APP_ID
+            - name: APP_KEY_PATH
+              value: /etc/monocle/app_key.rsa
+          image: quay.io/change-metrics/monocle_backend:latest
+          name: crawler-legacy
+          resources: {}
+          volumeMounts:
+            - mountPath: /etc/monocle
+              name: monocle-config-volume
+            - mountPath: /var/lib/crawler
+              name: crawler-dump-data-volume
+      restartPolicy: Always
```

Comment:
This process actually exit when there are no crawler defined, and the pod ends up in CrashLoopBackOff status. Not sure we can fixed that here, but we should make the process loop infinitely.